src/helpers/utilities.ts

```diff
@@ -469,6 +469,17 @@ export const omitFlatten = <T extends object, K extends keyof T>(
   return newObj;
 };
 
+export const chunk = <T>(input: T[], size: number): T[][] => {
+  return input.reduce<T[][]>((arr, item, idx) => {
+    return idx % size === 0
+      ? [...arr, [item]]
+      : [...arr.slice(0, -1), [...arr.slice(-1)[0], item]];
+  }, []);
+};
```

Comment:
I see we use this in the unique tokens stuff, which I can imagine might have a performance impact, but do we KNOW that it's affecting perf? This is a complex util, and without tests it makes me uneasy.